src/centos/scripts/start-marklogic.sh

```diff
@@ -67,8 +67,8 @@ echo "MARKLOGIC_INIT is defined, bootstrapping"
 
 curl --anyauth -i -X POST \
    -H "Content-type:application/x-www-form-urlencoded" \
-   --data-urlencode $MARKLOGIC_LICENSE \
-   --data-urlencode $MARKLOGIC_LICENSEE \
+   --data-urlencode $license-key=$LICENSE_KEY \
```

Comment:
Is there a reason there is a $ in front of license-key? It should be "license-key=$LICENSE_KEY" (include the quotes too)

Reply:
Yeah you're correct, I just tested the curl request and it fails with a parameter unauthorized error when the`$` is used. 

dd2cf38 
Updated.